### ISSUE TITLE
[Feature Request] Get Profile Description

### ISSUE THREAD

**Issue by teddymisty132** [sentanos/roblox-js-server#54]
Let it get the players Description. SO like 

Example code LUA

local message = api.GetAbout(message).message
print(message)

Module
blah blah blah
    http.Post(stuff stuff"/wqeqwqe?ewqjeqoweqehew?")
return blah blah
blah blah blah

SO yeah, I know you can get the blurb but bering able to get the description is better too.

**Comment by sentanos** [comment]
What? Isn't the blurb the same thing as the profile description?

**Comment by cvlver** [comment]
No. The Blurb is on the Top (Next to the Profile Picture) the description is The Status
